webapp/app.yaml

```diff
@@ -33,18 +30,7 @@ handlers:
 
 # Don't upload mock-data static files to AppEngine
 skip_files:
-- bower_components/accessibility-developer-tools/
-- bower_components/async/
-- bower_components/chai/
-- bower_components/lodash/
-- bower_components/mocha/
-- bower_components/sinon-chai/
-- bower_components/sinonjs/
-- bower_components/stacky/
-- bower_components/test-fixture/
-- bower_components/web-component-tester/
 - components/test/
-- node_modules/
```

Comment:
I'm a bit concerned about this. I understand we now need `node_modules` to include all the dependent components, but `node_modules` can contain a lot of stale dependencies & dev-only dependencies on our local machines.

I feel like we might want a new make rule to build release assets after all. And if so, we can use `polymer-build` instead of using the server-side rewriter. WDYT?

Reply:
Alternatively, we can use [`npm prune --production`](https://docs.npmjs.com/cli/prune.html) before deploying.

Reply:
So, polymer build is very specifically designed for an app that has a shell/entrypoint, and expects a range of things we don't do. `npm prune --production` on deploy sounds good; I'll use that for web_server.sh too.